KeyboardGuide.podspec

```diff
@@ -0,0 +1,21 @@
+Pod::Spec.new do |spec|
+  spec.name         = "KeyboardGuide"
+  spec.version      = "0.2.0"
+  spec.summary      = "A modern, real iOS keyboard system notifications handler framework that Just Works."
+  spec.description  = <<-DESC
+  A modern, real iOS keyboard system notifications handler framework that Just Works.
+  Based on years experience of iOS application development and various tests on each iOS version and device, it supports both Swift and Objective-C and works mostly reasonably on the latest 3 versions of iOS, which is iOS 11, 12 and iOS 13 now, and covers almost all iOS users.
+                   DESC
```

Comment:
I think this `description` is not required, as you mentioned in the comment it's shorten version of content that users can see on GitHub repository.